Core1Platform/02_CoreCapabilities.md

```diff
@@ -0,0 +1,51 @@
+# Core capabilities
+
+The *Actindo Core1 Platform* provides you with the following:
+- [Standalone solution or modernization of legacy ERP systems](./02_CoreCapabilities.md#standalone-solution-or-modernization-of-legacy-erp-systems)
+- [Omni-channel integration](./02_CoreCapabilities.md#omni-channel-orchestration)
+- [Data transformation and process orchestration](./02_CoreCapabilities.md#data-transformation-and-process-orchestration)
+- [Monitoring and reporting](02_CoreCapabilities.md#monitoring-and-reporting)
+ 
+
+## Standalone solution or modernization of legacy ERP systems
+
+The *Actindo Core1 Platform* supports the business of companies in wholesale and retail, sale via marketplaces or social commerce. It treats online commerce the same as stationary trading. Each store is its own marketplace, each transaction at a pay desk an incoming payment or refund.  
+The *Actindo Core1 Platform* is a complete interlocking solution. You can compose your individual platform by selecting only those modules you really need for your business.
+
+### ERP system is not yet in use
+
+If you have not yet an ERP system in use, the *Actindo Core1 Platform* can run as a standalone platform. The wide range of classic ERP features helps to digitize, automate, and optimize commerce processes. You can use it as all-in-one-solution to do the following:  
+    - Maintain your business data and do the accounting.   
```

Comment:
```suggestion
    - Maintain your business data and do the accounting.   
```
Was meinst du mit "do the accounting"? Das Buchhaltung Modul ist ziemlich alt und nicht aktualisiert worden, also denke ich nicht, dass die Kunden ihre Buchhaltung damit wirklich durchführen können. Sofern ich weiß, wird das Buchhaltung Modul eher als Nebenbuch verwendet, da die Daten der Bestellungen aus Fakturierung reinfließen, Kundendaten können gepflegt werden... aber dann werden die Daten exportiert und in andere Systemen überführt (wahrscheinlich SAP o.ä.).